src/main/java/com/konduto/sdk/models/KondutoBureauResponseField.java

```diff
@@ -0,0 +1,62 @@
+package com.konduto.sdk.models;
+
+public enum KondutoBureauResponseField {
+    EMAIL,
+    EMAIL_DOMAIN,
+    NAME,
+    EMAIL_DOMAIN_AGE,
+    EMAIL_FIRST_SEEN_DAYS,
+    EMAIL_FIRST_SEEN_AT,
+    EMAIL_LAST_SEEN_AT,
+    EMAIL_STATUS,
+    EMAIL_COUNTRY,
+    FRAUD_RISK,
+    REASON_FOR_ADVICE,
+    ADVICE,
+    RISK_BAND,
+    LAST_FLAGGED_AT,
+    DOB,
+    GENDER,
+    LOCATION,
+    SOCIAL_NETWORK_FRIENDS,
+    EMAIL_TOTAL_HITS,
+    EMAIL_UNIQUE_HITS,
+    EMAIL_EXISTS,
+    EMAIL_DOMAIN_EXISTS,
+    COMPANY,
+    TITLE,
+    EMAIL_DOMAIN_COMPANY,
+    EMAIL_DOMAIN_COUNTRY,
+    EMAIL_DOMAIN_CATEGORY,
+    EMAIL_DOMAIN_IS_CORPORATE,
+    EMAIL_DOMAIN_RISK_LEVEL,
+    EMAIL_DOMAIN_OTHER_INFO,
+    EMAIL_DOMAIN_RISKY_COUNTRY,
+    PHONE_OWNER_NAME,
+    PHONE_OWNER_TYPE,
+    PHONE_CARRIER_TYPE,
+    PHONE_CARRIER_NAME,
+    PHONE_OWNER_MATCH,
+    IP_RISK_LEVEL,
+    IP_RISK_REASON,
+    IP_REPUTATION,
+    IP_PROXY_TYPE,
+    IP_ISP,
+    IP_ORG,
+    IP_COUNTRY_CODE,
+    IP_REGION,
+    IP_CITY,
+    IP_RISK_SCORE,
+    EMAIL_AGE_DIGITAL_INDEX_SCORE,
+    EMAIL_IP_MATCH_CONFIDENCE,
+    EMAIL_PHONE_MATCH_CONFIDENCE,
+    EMAIL_BILLING_MATCH_CONFIDENCE,
+    EMAIL_SHIPPING_MATCH_CONFIDENCE,
+    IP_BILLING_MATCH_CONFIDENCE,
+    IP_SHIPPING_MATCH_CONFIDENCE,
+    PHONE_NAME_MATCH_CONFIDENCE,
+    PHONE_BILLING_MATCH_CONFIDENCE,
+    PHONE_SHIPPING_MATCH_CONFIDENCE,
+    NAME_BILLING_MATCH_CONFIDENCE,
+    NAME_SHIPPING_MATCH_CONFIDENCE;
+}
```

Comment:
@raphaelsampaio Pensei que poderia ser uma ideia melhor não hardcodar estes campos. Porque daí não precisamos lembrar de atualizar a SDK toda vez que algum campo novo for implementado.